### PR TITLE
SW-2117 Workflow to edit a planting site

### PR DESCRIPTION
**Pull request by constanzauanini**
Create Boundary component to reuse on Create and View components.

### CODE REVIEW

File: src/components/PlantingSites/BoundariesAndPlots.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import { Typography, Grid, Box, CircularProgress } from '@mui/material';
+import { Button, theme } from '@terraware/web-components';
+import strings from 'src/strings';
+import { TERRAWARE_SUPPORT_LINK } from 'src/constants';
+import { PlantingSite } from 'src/api/types/tracking';
+import { PlantingSiteMap } from '../Map';
+
+type BondariesAndPlotsProps = {
```
Comment (karthikbtf) [inline src/components/PlantingSites/BoundariesAndPlots.tsx:8]: typo, should be BoundariesAndPlotsProps?

**Review by karthikbtf — APPROVED** [review]
lgtm, one comment on typo. also post a screenshot if you can.

File: src/components/PlantingSites/BoundariesAndPlots.tsx (hunk 1 above)
Comment (constanzauanini) [reply src/components/PlantingSites/BoundariesAndPlots.tsx]: Thanks!

**Review by karthikbtf — APPROVED** [review]
looks good,  tried it in vercel. noticed that the map loads twice, unrelated to this PR.

**Pull request merged**